backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/UserService.java

```diff
@@ -197,6 +205,19 @@ public boolean verifyEmail(String userId, String code){
 		return true;
 	}
 
+	public User createUser(String userEmail, List<UserAttributeModel> userAttributesModel){
```

Comment:
I think this should be `public User createUser(User user)`. Also, service-layer classes should not know anything about web-layer models.

All service methods should accept only domain objects.